python/cudf/cudf/_lib/cpp/io/text.pxd

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2020-2021, NVIDIA CORPORATION.
+
+from libcpp.memory cimport unique_ptr
+from libcpp.string cimport string
+
+from cudf._lib.cpp.column.column cimport column
+
+
+cdef extern from "cudf/io/text/data_chunk_source.hpp" \
+        namespace "cudf::io::text" nogil:
+
+    cdef cppclass data_chunk_source:
+        data_chunk_source() except+
```

Comment:
It's possible that Cython is handling this correctly, but I've personally never seen `except+` without a space before. Unless you're certain it works, I'd recommend `except +` just to be safe; I've been bitten by subtle bugs in the Cython parser before. Same goes for the other instances in this file.
```suggestion
        data_chunk_source() except +
```